### ISSUE TITLE
Call Service Node Not Using msg. Object

### ISSUE THREAD

**Issue by mattivoxy** [zachowj/node-red-contrib-home-assistant-websocket#610]
### Describe the bug

I had a (for years) working flow where I would take the current temperature and forecast, and then generate a desired high/low temp range for my Ecobee. I would build a payload that the Ecobee would expect, and the thermostat would update its settings.

After upgrading to v11 of node-red, the data field of the service node no longer honors the injected msg.payload, and I get a "Call-service error. must contain at least one of temperature, target_temp_high, target_temp_low."

### To Reproduce

Node-Red function node / inject node, setting msg.payload = "{"data": {"target_temp_high": "77","target_temp_low": "70"}}" -> wire up to call service node - Domain: climate, service: set_temperature, entity: thermostat, data: <blank>

If I manually place the payload in the data field of the call service node, it works. But it seems that the service node no longer accepts the injected msg.payload.data object into the data field.

### Expected behavior

I would expect this to work as it always has, and the msg.payload.data object would be populated into the data field of the service node.

### Screenshots

![image](https://user-images.githubusercontent.com/67348366/156041721-213dd971-a2ff-4743-a10a-1d75f1701644.png)
![image](https://user-images.githubusercontent.com/67348366/156041764-73a6f997-f672-46da-8e70-cab6bc7d79f7.png)
![image](https://user-images.githubusercontent.com/67348366/156041791-66f7e7c2-49f6-4091-be3a-50ee4582b728.png)

### This package's version (not the Home Assistant add-on version)

0.43.0

### Is Node-RED running in Docker?

Home Assistant add-on

### Node-RED version

2.2.2

### Node.js version

16.14.0

**Comment by zachowj** [comment]
The input of the call-service node was never intended to accept a string as the `payload`. It only accepted them before due to a bug fixed in 0.42.0.

https://zachowj.github.io/node-red-contrib-home-assistant-websocket/node/call-service.html#data-2

Change `msg.payload` to JS object.

```
msg.payload = { 
  data: {
    target_temp_high: targetHight,
    target_temp_low: targetLow,
  }
};
```

**Comment by mattivoxy** [comment]
MUCH appreciated. Guess I was taking advantage of a long-standing bug for quite some time!
